legal-api/src/legal_api/services/__init__.py

```diff
@@ -33,7 +33,7 @@
 from .bootstrap import AccountService, RegistrationBootstrapService
 from .business_details_version import VersionedBusinessDetailsService
 
-# from .queue import QueueService
+from .queue import QueueService
```

Comment:
Having this commented caused an import error on pytest collection (specifically, in `test_internal.py`). With the GCP changes I am not sure if this will eventually be removed, but for now I think it should still be exported from this module unless we just want to remove the tests in `test_internal.py`.